### PR TITLE
Support configuring distro aliases via ENV and set aliases in the composer template.

### PR DESCRIPTION
**Pull request by thozza**
SSIA, check commit messages for more details.

Also added unit tests for the functionality.

### CODE REVIEW

**Comment by thozza** [comment]
@croissanne I'm not 100% sure that my approach with the template is correct and sufficient. Please let me know if I need to do it differently or if I missed some place that required modification. Thanks!

File: cmd/osbuild-composer/config.go [hunk 1]
```diff
@@ -14,7 +15,7 @@ type ComposerConfigFile struct {
 	Koji          KojiAPIConfig     `toml:"koji"`
 	Worker        WorkerAPIConfig   `toml:"worker"`
 	WeldrAPI      WeldrAPIConfig    `toml:"weldr_api"`
-	DistroAliases map[string]string `toml:"distro_aliases"`
+	DistroAliases map[string]string `toml:"distro_aliases" env:"DISTRO_ALIASES"`
```
Comment (bcl) [inline cmd/osbuild-composer/config.go:18]: This should have an `OSBUILD_` prefix. Just in case.

Comment (thozza) [reply cmd/osbuild-composer/config.go:18]: I considered it, but then saw that the other configuration options also do not use any prefix. :shrug:

Comment (bcl) [reply cmd/osbuild-composer/config.go:18]: Odd. When I added `OSBUILD_LOCALSAVE` that was one of the changes folks requested. In practice it likely doesn't really matter.

**Review by croissanne — APPROVED** [review]
thank you!

**Review by ondrejbudai — APPROVED** [review]

**Pull request merged**